sdk/monitor/monitor-opentelemetry-exporter/src/export/base.ts

```diff
@@ -93,11 +104,25 @@ export abstract class AzureMonitorBaseExporter {
    */
   protected async _exportEnvelopes(envelopes: Envelope[]): Promise<ExportResult> {
     diag.info(`Exporting ${envelopes.length} envelope(s)`);
+    
+    // Shutdown statsbeat if the maximum number of failures is exceeded
+    if (
+      this._statsbeatFailureCount > MAX_STATSBEAT_FAILURES &&
+      this._statsbeatMetrics &&
+      this._statsbeatMetrics.isEnabled()
+    ) {
+      this._statsbeatMetrics.enable(false);
+      this._statsbeatMetrics.shutdown();
+    }
+    
     if (envelopes.length < 1) {
       return { code: ExportResultCode.SUCCESS };
     }
+    
     try {
+      const startTime = Number(new Date().getTime());
```

Comment:
Date().getTime() should always be number of milliseconds right? do you need the Number cast?